sdk/eventhub/azure-eventhub-checkpointstoretable/azure/eventhub/extensions/checkpointstoretable/_tablestoragecs.py

```diff
@@ -2,8 +2,37 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
+import datetime
+import time
+import logging
+import calendar
+import dateutil.parser
+from azure.eventhub._eventprocessor.checkpoint_store import CheckpointStore
+from azure.data.tables import TableClient, UpdateMode
+from azure.data.tables._base_client import parse_connection_str
```

Comment:
@swathipil I'm just curious, but what is the reason to vendor tables rather than take a dependency on it? I don't fully understand why that was done for storage-blob.

Reply:
That's a good question that I don't know the answer to :P
In storage-blob, it was to fix this issue: https://github.com/Azure/azure-sdk-for-python/pull/9950
I'll start an email thread to ask about taking a dependency on tables.